.travis.yml

```diff
@@ -12,16 +24,7 @@ matrix:
     env: TOX_ENV=py36
```

Comment:
With `tox-travis` you don't need these `TOX_ENV=` statements

Reply:
So the test matrix is invoked by the tox command and that's it? Wasn't sure about the travis config at all, I do prefer using GitHub Actions for these cases 🙈

Reply:
@jkimbo Looks like removing the matrix setting didn't allow travis to run on several python env...